routes/ffxiv.py

```
@@ -46,3 +46,54 @@
+            "realmName",
+            "realmNames",
+            "link",
+        ]
+        response = [{key: item.get(key) for key in column_order} for item in response]
+        fieldnames = list(response[0].keys())
+
+        return return_safe_html(
+            render_template(
+                "petshoppinglist.html", results=response, fieldnames=fieldnames, len=len
+            )
+        )
```

Ruff (0.8.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)